src/TConsole.cpp

```diff
@@ -267,6 +268,14 @@ void TConsole::Command_Clear(const std::string&, const std::vector<std::string>&
     mCommandline->write("\x1b[;H\x1b[2J");
 }
 
+void TConsole::Command_Version(const std::string& cmd, const std::vector<std::string>& args) {
+    if (!EnsureArgsCount(args, 0)) {
+        return;
+    }
+
+    mCommandline->write("Current version: " + Application::ServerVersionString());
```

Comment:
For consistency, we could use `Application::Console().WriteRaw()` or just `WriteRaw()` instead... Not sure why the other functions are inconsistent about the use of this as well.

Maybe just changing this to use `Application::Console().WriteRaw()` might be nice, to at least be consitent with most of the other commands. A separate issue can be opened to address this inconsistency and the weird way in which it's called (via Application::?? why???? who knows.)